test/fixtures/command-workflow/features/commands.feature

```diff
@@ -0,0 +1,77 @@
+Feature: Tests the command servlet works as expected
+
+  Scenario: Commands are received in order
+    Given I add a command with message "first"
+    And I add a command with message "second"
+    Then I run the "bounce_command" test script
+    And I wait to receive an error
+    Then the error payload field "command_response.message" equals "first"
+    And the error payload field "command_response.uuid" is stored as the value "uuid"
+    And the error payload field "command_response.run_uuid" is stored as the value "run_uuid"
+    And I discard the oldest error
+    Then I run the "bounce_command" test script
+    And I wait to receive an error
+    Then the error payload field "command_response.message" equals "second"
+    And the error payload field "command_response.uuid" does not equal the stored value "uuid"
+    And the error payload field "command_response.run_uuid" equals the stored value "run_uuid"
+
+  Scenario: Idempotent commands are sent correctly
+    Given I generate a series of commands with sequential UUIDs
+    Then I run the "bounce_idempotent_command" test script with UUID "1"
+    And I wait to receive an error
+    Then the error payload field "command_response.message" equals "second"
+    And the error payload field "command_response.uuid" equals "2"
```

Comment:
I'm thinking you could discard the oldest error and then repeat all of these lines to prove that we really do get the same thing again for a second call with the same UUID?